server/bin/unittests

```diff
@@ -261,7 +261,7 @@ function _setup_state {
     let res=res+$?
     cp -a $_tdir/../../lib/* $_testopt/common/lib
     let res=res+$?
-    cp -a $_tdir/../../bin/* $_testopt/common/bin
+    cp -a $_tdir/getconf.py $_testopt/common/bin
```

Comment:
Why do we need this when `getconf.py` lives in `server/bin/`?

Reply:
yeah. I removed it with other changes